### DISCUSSION TITLE
Game not launching

### DISCUSSION THREAD

**Discussion by sallymanda** [Kron4ek/minecraft-vortex-launcher#31] [Q&A] [answered]
Hey all, just trying to get vortex launching minecraft and no success so far. Using v1.1.15 on bullseye, java installed and updated, missing libraries set to auto update... not sure what I'm doing wrong :confused:

**Comment by Kron4ek** [comment, marked as answer]
Hi, what Minecraft version are you trying to run and what Java version are you using? Also, are there any crash-reports inside the game directory?

**Reply by sallymanda** [reply]
Oh oops, forgot to mention what version number! Well 1.16 works fine, but I'm trying to launch 1.17.1, and no other versions I've downloaded so seem to work either. As far as I can tell it's only generating logs for successful launching of 1.16...

**Reply by Kron4ek** [reply]
Ok, so what about Java version? You can find it out by executing the `java -version` command in terminal.

**Reply by sallymanda** [reply]
Oh oops forgot java version, sorry! :woman_facepalming:

`openjdk version "11.0.13" 2021-10-19
OpenJDK Runtime Environment (build 11.0.13+8-post-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 11.0.13+8-post-Debian-1deb11u1, mixed mode, sharing)
`

**Reply by sallymanda** [reply]
Okay I used a custom java directory and it works now :confused: is the other java (above) the wrong version?

**Reply by Kron4ek** [reply]
Your default Java version is just too old, new Minecraft versions (1.17 and newer) require at least Java 16. Note, however, that older Minecraft versions (for example, 1.12.2) may not work with Java 16+.

If you have multiple Java versions installed (say, openjdk-11 and openjdk-17), you can choose the default version using the `update-java-alternatives` command.

**Reply by sallymanda** [reply]
Thanks! Such an easy solution :sweat_smile: I removed 11 and updated to 17 as I don't need to use older versions for now so not a big deal :relaxed:

Buuut, I accidentally discovered what I think *may* be a bug: the path to my installation contains a space (its not stored in the standard `~/.minecraft` directory) and when I launch minecraft via vortex the game saves my worlds to a path that is truncated at the point where the space is. For example `~/Games/Linux Games/Minecraft/saves` becomes `~/Games/Linux/saves`. When I launch minecraft using the exsting start.sh (without vortex) the worlds save in the correct save directory, so seems like it's vortex... Any suggestions?

**Reply by Kron4ek** [reply]
You are right, it is a bug in the launcher. There is nothing you can do about it, except removing spaces from the path to the game.

**Reply by sallymanda** [reply]
Ah damn, is it something that will be fixed in future versions, or is it hard baked into it?

**Reply by Kron4ek** [reply]
Yes, it it will likely be fixed in the next version, i don't think it is hard to fix it.

**Reply by sallymanda** [reply]
Oh great :relaxed: well thanks for the support, and thanks for making vortex in the first place, love it that there's something like this for linux users :smile:
